app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -278,8 +278,8 @@ def create_vm(args = {})
       vm = super({ :first_boot_dev => 'network', :quota => ovirt_quota }.merge(args))
 
       begin
-        create_interfaces(vm, args[:interfaces_attributes])
-        create_volumes(vm, args[:volumes_attributes])
+        create_interfaces(vm, args[:interfaces_attributes]) unless args[:interfaces_attributes].empty?
```

Comment:
Wouldn't the check be better to put at the beggining of the `create_interface` method?

Reply:
I don't mind, but can you explain why it would be better?

Reply:
In case the method gets called from a different place :) but it's not a nack.